Formula/h/hdf5-mpi.rb

```diff
@@ -20,54 +25,46 @@ class Hdf5Mpi < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "2d3d1ccbf33cbbe10e13a6969d682217e650eefbaecc0f4a2e62c79394ce06a7"
   end
 
-  depends_on "autoconf" => :build
-  depends_on "automake" => :build
-  depends_on "libtool" => :build
+  depends_on "cmake" => :build
   depends_on "gcc" # for gfortran
   depends_on "libaec"
   depends_on "open-mpi"
+  depends_on "pkg-config"
```

Comment:
Is this needed at runtime?

Reply:
I don't think so. But it's the same in in `hdf5.rb`. I will remove it there as well.

Reply:
yes, the wrappers need it.